server/services/licenceSearchService.js

```diff
@@ -15,14 +15,14 @@ module.exports = function createLicenceSearchService(
   nomisClientBuilder,
   prisonerSearchApi
 ) {
-  const bookingIdForExistingLicence = async (bookingId) => {
-    const licence = await licenceClient.getLicence(bookingId)
+  const bookingIdForLicenceIncludingSoftDeleted = async (bookingId) => {
```

Comment:
renamed to make it clear that search results could include soft deleted licences